microscopium/preprocess.py

```diff
@@ -910,7 +910,7 @@ def create_missing_mask(missing, order, rows=512, cols=512):
     return mask
 
 
-def montage_with_missing(fns, order=None):
+def montage_with_missing(fns, *, order, re_string, re_group):
     """Montage a list of images, replacing missing fields with dummy values.
```

Comment:
What are the dummy values, NA?

Reply:
No, dummy = 0 at this point. Anything else will wreak havoc with the feature computation. The function also returns a mask but currently we can't use it.

Reply:
> The function also returns a mask but currently we can't use it.

Uh, why?

Reply:
I meant, we don’t have any functions (specifically feature computation functions) that can take in masks as input.